_extras/checkout.md

```diff
@@ -105,6 +102,21 @@ The lessons you may submit to are:
 3.  [Stable Software Carpentry lessons](https://software-carpentry.org/lessons/):
     we are currently accepting solutions to existing exercises, diagrams, and improvements to learning objectives.
 
+> ## What Makes a Good Exercise?
+>
+> 1.  Follow the coding style of the lesson you're submitting to.
```

Comment:
At least for Data Carpentry, the repositories have a CONTRIBUTING file that lists the style guidelines for the lessons. Could you maybe add a mention to it?